controllers/drplacementcontrol_controller.go

```diff
@@ -656,6 +704,17 @@ func (d *DRPCInstance) runInitialDeployment() (bool, error) {
 			d.drpcPlacementRule)
 	}
 
+	// We are done if the initial deployment is already complete
+	if len(d.userPlacementRule.Status.Decisions) > 0 {
+		d.log.Info(fmt.Sprintf("Already deployed to %s. Last state %s",
+			d.userPlacementRule.Status.Decisions[0].ClusterName, d.getLastDRState()))
+
+		return done, nil
```

Comment:
`sanityCheckDelay` will not trigger VRG status as `needStatusUpdate` will not be set?

Reply:
Good catch.  I meant to add `LastUpdateTime` and set `needStatusUpdate` if since the last update time is greater than the `sanityCheckDelay`.  We'll address it next.